views/databaseIssue.ejs

```diff
@@ -100,6 +100,17 @@
                             disabled>
                         Save connection setting
                     </button>
+                    <button
+                            type="button"
+                            id="restart-server-button"
+                            class="btn btn-danger"
+                            onclick="postRestartServer()"
+                    <% if (isDocker) { %>
+                            hidden
+                            <% } %>
+                            disabled>
+                        Restart OctoFarm
+                    </button>
```

Comment:
New restart server button. Hidden for docker as this page won't be able to save anyway and the servers already restarted by the time they update their environment